src/style.css

```diff
@@ -1 +1,69 @@
-/* add styles here */
+@font-face {
+  font-family: Avenir;
+  src: local(Avenir) url(fonts/Avenir-Book.ttf) format('truetype');
```

Comment:
In the '@font-face' rule, the 'src' property should separate local fonts and URLs with a comma. Correct it to 'src: local(Avenir), url(fonts/Avenir-Book.ttf) format('truetype');' to ensure proper font loading.